python/plotting/configs/cutstrings.py

```diff
@@ -16,20 +16,27 @@ def baseline(channel, cut_type):
 		cuts["os"] = "((q_1*q_2)<0.0)"
 		
 		if channel == "mm":
+			cuts["trg"] = "(trg_singlemuon == 1)"
 			cuts["extra_lepton_veto"] = "(extraelec_veto < 0.5)*(extramuon_veto < 0.5)"
 			cuts["iso_1"] = "(iso_1 < 0.15)"
 			cuts["iso_2"] = "(iso_2 < 0.15)"
 			cuts["m_vis"] = "(m_vis > 60.0)*(m_vis < 120.0)"
 		elif channel == "ee":
-			pass
+			cuts["trg"] = "(trg_singleelectron == 1)"
+			cuts["extra_lepton_veto"] = "(extraelec_veto < 0.5)*(extramuon_veto < 0.5)"
+			cuts["iso_1"] = "(iso_1 < 0.1)"
+			cuts["iso_2"] = "(iso_2 < 0.1)"
+			cuts["m_vis"] = "(m_vis > 60.0)*(m_vis < 120.0)"
 		elif channel == "em":
+			cuts["trg"] = "(trg_muonelectron == 1)"
 			cuts["trigger_threshold"] = "(pt_1 > 24.0 || pt_2 > 24.0)" if "2016" in cut_type else "(1.0)"
 			cuts["pzeta"] = "(pZetaMissVis > -35.0)" if "2016" in cut_type and not "mssm" in cut_type else "(pZetaMissVis > -20.0)"
 			cuts["extra_lepton_veto"] = "(extraelec_veto < 0.5)*(extramuon_veto < 0.5)"
 			cuts["iso_1"] = "(iso_1 < 0.15)"
 			cuts["iso_2"] = "(iso_2 < 0.2)" if "2016" in cut_type else "(iso_2 < 0.15)"
 			#if not "mssm" in cut_type: cuts["bveto"] = "(nbtag == 0)"
 		elif channel == "mt":
+			cuts["trg"] = "(trg_singlemuon == 1)"
```

Comment:
Corrected:

https://github.com/cms-analysis/HiggsAnalysis-KITHiggsToTauTau/commit/b037c4914d639d5c808d14a3eb60f3d4a361fce4

Do you plan to switch to the inverted diTau-Pair trigger logic in SM, Alex?